### DISCUSSION TITLE
Mifare Ultralight AES Suppliers?

### DISCUSSION THREAD

**Discussion by MineTech13** [stustapay/stustapay#388] [Q&A] [answered]
I am currently investigating this system and conducting experiments and tests. I have come to a point where I would like to conduct a small test using real NFC Chips in my research. However, I am facing difficulties in finding a supplier for the Mifare Ultralight AES tags that does not require ordering in large quantities. Can you provide any recommendations for suppliers that offer smaller amounts?

**Comment by TheJJ** [comment, marked as answer]
we order them in ~15k quantities, so smaller amounts may be difficult. you may get test samples from https://www.ikastetikett.de/ or other nfc suppliers. how many do you need for what use-case?

**Reply by MineTech13** [reply]
Well we are kinda just starting so we expect numbers sub 1k for the beginning thooo this system sounds very interesting and useful we want to use it for a convention basically ^^

**Reply by TheJJ** [reply]
it certainly is suitable :)  please keep us up to date, we're glad when other events can make use of our work

**Reply by tonowie** [reply]
How can I tell that the band on the https://www.ikastetikett.de/ site is compatible? I see no mentions of Mifare Ultralight AES there. Specs on the site mention following but neither reminds me of the information from guides
> ICODE SLI, ICODE SLIX, ICODE SLI-S, Mifare Ultralight EV1, Mifare Ultralight, NTAG216, Mifare 1k S50, FM1108, Ntag203, Ntag213, Alien H3(9613), Alien H3(9629), EM4200, TK4100

**Reply by TheJJ** [reply]
when you talk to the sales people, they can select this chip. but there's plenty of other suppliers in the world.

**Comment by 2Turbulent** [comment]
Do you have other shop suggestions?
As consumer it is difficult to find these Mifare Ultralight AES. Can other type of cards also be used the 1k classic mifare for example?

**Reply by mikonse** [reply]
To find suppliers, just research the Internet for your local region.
Other chip types are not yet supported, but adding them should be possible rather easily.
We would recommend to not use the Mifare Classic due to its grave security flaws.

If you want to add support for other chip types we are happy to accept contributions.
